helmfile.d/20-dashboard.yaml

```diff
@@ -0,0 +1,150 @@
+bases:
```

Comment:
I suggested merging the helmfiles files together but now it's difficult to me to know which releases should be installed in which case, can we add more documentation about dependencies of each component?

Reply:
Or maybe sub group the apps in the same file and keep the ones that are related close to each other

Reply:
I added comments to the relevant sections in _base.yaml_. Do you think it is ok like this?

Reply:
Yes it is good now, thanks!